### ISSUE TITLE
Content from Authenticated Exp Team 2

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#31186]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more new unauthenticated (static) VA.gov pages
- [ ] Making revisions to one or more existing unauthenticated (static) VA.gov page
- [ ] Launching a new online form, app, or tool on VA.gov
- [X] Making revisions to an existing online form, app, or tool on VA.gov
- [ ] Something else (please add details in the next question)

## What do you need help with?
We are adding several new fields to the authenticated profile and need help choosing appropriate language for field labels and options.  
  
## Supporting Artifacts
List of fields being added.  We need Content to provide recommendations on the Label field and Pre-Populated Options, along with the proper way to label the "Other" text field.

| New Field | Label              | Form Control                                        | Max Length | Data Type | Pre-Populated Options                                                                                                                                        | Comments                                                             |
|-----------|--------------------|-----------------------------------------------------|------------|-----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------|
| 1         | Preferred name     | Text input                                          | 25         | Char      | N/A                                                                                                                                                          | Also mockup state with Other selected                                |
| 2         | Pronouns           | Checkboxes + text input (2a) when "Other" selected  | Pre-filled | Char      | "He/him/his", "She/her/hers", "They/them/theirs", "Ze/zir/zirs", "Patient Name", "Another Option:"                                                           | Also mockup read-only state showing how mutiple options will display |
| 2a        | Other              | Text input                                          | TBD        | Char      | N/A                                                                                                                                                          | Captures custom value for Pronouns when Other is selected            |
| 3         | Gender identity    | Select box + text input (3a) when "Other" selected  | Pre-filled | Char      | “Male", "Female", "Transgender Male", "Transgender Female", "Non-Binary", "Does not wish to disclose", "Another Option:"                                     |                                                                      |
| 3a        | Other              | Text input                                          | TBD        | Char      | N/A                                                                                                                                                          | Captures custom value for Gender Identity when Other is selected     |
| 4         | Sexual orientation | Select box + text input (4a) when "Other" selected  | Pre-filled | Char      | TBD "Lesbian", gay or homosexual, Straight or heterosexual, Bisexual, Queer, Something else, please describe (Open text), Don’t know, choose not to disclose |                                                                      |
| 4a        | Other              | Text input                                          | TBD        | Char      | N/A                                                                                                                                                          | Captures custom value for Sexual Orientation when Other is selected  |

- Link to product outline: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/personal-information/preferred-name-pronouns-gender-sexual-orientation/README.md
- Link to or attach user flows: 
- Link to prototype or mockups:  https://www.sketch.com/s/ba254d92-3c3d-4eba-825d-d7f5bda35565
- Link to any epics/issues that may be helpful: #31103

## Will this work be going through the Collaboration Cycle?
- [X] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date:  TBD; there are BE dependencies
- Estimated staging review date:  TBD
- Content and IA work needed by: TBD

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [ ] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: 
-	OCTO-DE product owner: Samara Strauss
-	Product manager: Marci McGuire
-	Designer: Kevin Hoffman
-	FE engineer: Zach Morel
-	Product/team Slack channel: vsa-authd-exp

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#va-sitewide-content](https://dsva.slack.com/channels/va-sitewide-content) Slack channel and tag @RLHecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by MarciMcGuireGCIO** [comment]
Added a comment to field table regarding the use of "Patient Name".

**Comment by DanielleThierryUSDSVA** [comment]
Thanks, @MarciMcGuireGCIO! Just to clarify - those questions aren't meant to be answered now. I was just noting the things I'm curious to find out about through your team's usability research!

**Comment by mnorthuis** [comment]
@MarciMcGuireGCIO When research is complete and mocks and content are a bit more finalized, we can officially determine the URL for the new page.  This one seems pretty straight forward (i.e. /contact-information), but just want to see how the experience and testing shakes out first.  Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @Samara-Strauss @MarciMcGuireGCIO. I realized I didn't answer the question about changing "sex assigned at birth" to "birth sex" to make the field name shorter. I want to just clarify that there is a difference in meaning between these terms. "Birth sex" implies that you were born a specific sex. "Sex assigned at birth" specifically acknowledges that a doctor assigned you a sex at birth and that may not have actually been correct (for example, in the case of someone who is intersex and was assigned male or female).

**Comment by MarciMcGuireGCIO** [comment]
Thanks, @DanielleThierryUSDSVA.   @Samara-Strauss since the MPI uses "Birth sex" and the mockups have already been completed using "Birth sex", should we leave as-is and address during user research? cc @cgednalske

**Comment by Samara-Strauss** [comment]
@MarciMcGuireGCIO it sounds like @DanielleThierryUSDSVA's final decision is to use Sex assigned at birth. Is that correct, @DanielleThierryUSDSVA? If that's the case, we should update all mocks with "Sex assigned at birth" and test with that copy.

**Comment by MarciMcGuireGCIO** [comment]
Thanks, @Samara-Strauss!  I'll create a task for Christina to update the mockups.

**Comment by DanielleThierryUSDSVA** [comment]
@Samara-Strauss That's the terminology we're using for the 10-10EZ ("What sex were you assigned at birth?") so I think it's good to test with that same terminology here...and this will also help us understand if "sex assigned at birth" is confusing for folks. I do want to acknowledge that as with all language in this space, the terminology is evolving and there isn't a lot of consensus yet around what is definitively the right way to go. I want to make sure to point out the nuances in the terminology so the team is aware, but I also want to be sure to note that I don't know that there is a definitive "right" here. THanks!

**Comment by Samara-Strauss** [comment]
Thanks, @DanielleThierryUSDSVA! Sounds like you're already on it @MarciMcGuireGCIO, but we should update all mocks and test with "Sex assigned at birth" per Danielle's guidance.

**Comment by MarciMcGuireGCIO** [comment]
Yep, I'm on it! Thank @DanielleThierryUSDSVA and @Samara-Strauss !
